Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM php:7.2-fpm
+
+LABEL version="1.0.0"
+LABEL description="Postuf Telegram OSINT Library"
+
+RUN apt-get update && apt-get install -y git \
+zlib1g-dev \
+libicu-dev \
+libgmp-dev \
+g++ \
+libzip-dev \
+unzip \
+&& docker-php-ext-install zip \
+&& docker-php-ext-configure gmp \
+&& docker-php-ext-install gmp \
+&& docker-php-ext-install sockets
```

Comment:
RUN docker-php-ext-install sockets
RUN docker-php-ext-install pcntl
RUN docker-php-ext-install bcmath
RUN docker-php-ext-install gmp

Reply:
@anton-postuf current spec seems legit, we don't use pcntl/bcmath anywhere.